frontend/src/components/CreateDataSourceExample.vue

```diff
@@ -159,7 +159,7 @@ const grantStatement = (
     switch (engineType) {
       case "MYSQL":
       case "TIDB":
-        return "CREATE USER bytebase@'%' IDENTIFIED BY 'YOUR_DB_PWD';\n\nGRANT ALTER, ALTER ROUTINE, CREATE, CREATE ROUTINE, CREATE VIEW, \nDELETE, DROP, EVENT, EXECUTE, INDEX, INSERT, PROCESS, REFERENCES, \nSELECT, SHOW DATABASES, SHOW VIEW, TRIGGER, UPDATE, USAGE \nON *.* to bytebase@'%';";
+        return "CREATE USER bytebase@'%' IDENTIFIED BY 'YOUR_DB_PWD';\n\nGRANT ALTER, ALTER ROUTINE, CREATE, CREATE ROUTINE, CREATE VIEW, \nDELETE, DROP, EVENT, EXECUTE, INDEX, INSERT, PROCESS, REFERENCES, \nSELECT, SHOW DATABASES, SHOW VIEW, TRIGGER, UPDATE, USAGE, \nFLUSH_TABLES, LOCK TABLES, REPLICATION CLIENT, REPLICATION SLAVE, \nREPLICATION_APPLIER, SESSION_VARIABLES_ADMIN \nON *.* to bytebase@'%';";
```

Comment:
Make sure AWS RDS/GCP CloudSQL users can create the user in this way.

Reply:
IMO it's a whole different story.
We would sort out the privileges for AWS RDS/GCP CloudSQL later, and this change is only for a prompt when creating a user with minimal privileges on-prem.

Reply:
Cloud is the future, PERIOD :-) If this doesn't work, we have to figure out other solutions/designs.

Reply:
Just tried, users can get privileges above.